docker/config/local_dev.env

```diff
@@ -46,20 +46,3 @@ OIDC_OP_USER_ENDPOINT=http://oidcprovider:8080/openid/userinfo
 OIDC_VERIFY_SSL=false
 # Disable NotBlockedInAuth0Middleware
 ENABLE_AUTH0_BLOCKED_CHECK=false
-
-
-# Eliot
```

Comment:
These are vestigates of when Eliot was in this repo that should have been removed ages ago.